specifyweb/frontend/js_src/lib/components/DataModel/types.ts

```diff
@@ -6791,10 +6798,10 @@ export type TectonicUnit = {
   readonly toOneIndependent: {
     readonly acceptedTectonicUnit: TectonicUnit | null;
     readonly createdByAgent: Agent | null;
+    readonly definition: TectonicUnitTreeDef;
+    readonly definitionItem: TectonicUnitTreeDefItem;
     readonly modifiedByAgent: Agent | null;
     readonly parent: TectonicUnit;
-    readonly tectonicUnitTreeDef: TectonicUnitTreeDef;
-    readonly tectonicUnitTreeDefItem: TectonicUnitTreeDefItem;
   };
   readonly toManyDependent: RR<never, never>;
   readonly toManyIndependent: RR<never, never>;
```

Comment:
Does TectonicUnit need `children` and `acceptedChildren` to-many independent fields like other trees?

Reply:
it's not in the requirements but it's maybe an oversight

Reply:
Definitely need them: You wouldn't be able to merge or synonymize currently if the trees don't have `children` and `acceptedChildren`

Reply:
I see. We'll have to change the model for that, which means... yay more migrations 😭